packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -538,7 +538,7 @@ contract Bridge is EssentialContract, IBridge {
         returns (bool success_, uint32 numCacheOps_)
     {
         try ISignalService(_signalService).proveSignalReceived(
-            _chainId, resolve(_chainId, "bridge", false), _signal, _proof
+            _chainId, resolve(_chainId, LibStrings.B_BRIDGE, false), _signal, _proof
```

Comment:
could you please also replace ""signal_service" in SignalSerivce.sol?

Reply:
Also in vaults, there are:

```
/// @inheritdoc BaseVault
    function name() public pure override returns (bytes32) {
        return "erc1155_vault";
    }
``` 

You may also help me replace the name with LibStrings.xxx